.golangci.yml

```diff
@@ -4,7 +4,7 @@ run:
 linters:
   enable:
     - bodyclose
-    - depguard
+     # - depguard as of v1.54.2, the default config throws errors on our repo
```

Comment:
Is it worth fixing them?

Reply:
Don't think so, we already have good import linting

Reply:
And also, didn't use it before. It whitelisted everything